src/main/java/uk/gov/companieshouse/presenter/account/kafka/KafkaMessageHelper.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.companieshouse.presenter.account.kafka;
+
+import java.time.ZoneOffset;
+import java.util.Objects;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.producer.CHKafkaProducer;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.presenter.account.exceptionhandler.KafkaMessageFailException;
+import uk.gov.companieshouse.presenter.account.exceptionhandler.KafkaMessageInterruptedException;
+import uk.gov.companieshouse.presenter.account.serialiser.PresenterCreatedSerialiser;
+import uk.gov.companieshouse.presenter.account.utils.DateTimeGenerator;
+import uk.gov.companieshouse.presenter_account.PresenterCreated;
+
+@Component
+public class KafkaMessageHelper {
+
+    private static final String KAFKA_INTERRUPT_MESSAGE = "Kafka message interrupted while senting";
+    private static final String ERROR_MESSAGE = "Failed to send Kafka message";
+
+    private final CHKafkaProducer kafkaProducer;
+
+    private final DateTimeGenerator dateTime;
+
+    private final PresenterCreatedSerialiser serialiser;
+
+    private final Logger logger;
+
+    @Autowired
+    public KafkaMessageHelper(
+            CHKafkaProducer kafkaProducer,
+            DateTimeGenerator dateTime,
+            PresenterCreatedSerialiser serialiser,
+            Logger logger) {
+        this.dateTime = dateTime;
+        this.kafkaProducer = kafkaProducer;
+        this.logger = logger;
+        this.serialiser = serialiser;
+    }
+
+    public PresenterCreated createPresenterCreated(final String presenterAccountDetailId) {
+        Objects.requireNonNull(presenterAccountDetailId);
```

Comment:
I don't think you need all these Objects.requireNonNull calls in this class. We know we aren't sending null values.